.vscode/cspell.json

```diff
@@ -1364,6 +1364,12 @@
         "protobuf"
       ]
     },
+    {
+      "filename": "**/sdk/search/**/*.cs",
+      "words": [
+        "agentic"
+      ]
+    },
```

Comment:
This [filename](https://github.com/Azure/azure-sdk-for-net/blob/de99c669d49e35bbb79ce3100271c980180c15a3/.vscode/cspell.json#L1485) already exists. Can we move this word under that?